README.md

```diff
@@ -0,0 +1,62 @@
+# Data Modeling with Postgres
+## Sparkify data analysis
+ *Nastaran Fazeli*
+
+The objective of this project is to build an ETL pipeline using ***Python***, ***PostgresSQL*** . We want to understand what songs users are listening to using an easy way to query the data, which resides in a directory of JSON logs (log_data), as well as a directory with JSON metadata (song_data) on the songs.
+
+I to created a postgres database schema and an ETL pipeline for this analysis. 
+
+## Song Dataset
+The first dataset is a subset of real data from the [Million Song Dataset](http://millionsongdataset.com/). Each file is in JSON format and contains metadata about a song and the artist of that song. The files are partitioned by the first three letters of each song's track ID.
+
+## Log Dataset
+The second dataset consists of log files in JSON format generated by this [event simulator](https://github.com/Interana/eventsim)  based on the songs in the song dataset. These simulate activity logs from a music streaming app based on specified configurations.
+
+The log files in the dataset were partitioned by year and month. e.g. log_data/2018/11/2018-11-12-events.json
+
+
+## Star Scheme
+The image shows the star scheme used to model the Sparkify database in the cente is the ***fact table***:
+* ***songplay*** - records in log data associated with song plays i.e. records with page NextSong, <br>
+
+surounded by the ***dimention tables***:
+ 1. ***users*** - users in the app
+ 2. ***songs*** - songs in music database
+ 3. ***artists*** - artists in music database
+ 4. ***time*** - timestamps of records in ***songplays*** broken down into specific units
+
+ ![Tux, Star_scheme](/images/Star_scheme.png)
+
+ ## Repo files
+ 1. test.ipynb displays the first few rows of each table to let us check the database.
+2. create_tables.py drops and creates the tables. We run this file to reset the tables before each time we run the ETL scripts.
+3. etl.py reads and processes files from song_data and log_data and loads them into the tables.
+4. sql_queries.py contains all the sql queries, and is imported into the last three files above.
+5. README.md what you are reading now.
+
+## Final tables (first five rows):
+***users***
+
+ ![Tux, users](/images/users.png)
```

Comment:
Github seems to be case-sensitive (change the file ending of the screenshots to lower case).